lib/PuppeteerSharp/Launcher.cs

```diff
@@ -125,7 +125,7 @@ public async Task<Browser> LaunchAsync(LaunchOptions options)
                     _logger.LogInformation("Process Count: {ProcessCount}", Interlocked.Increment(ref _processCount));
                 }
 
-                var browser = await Browser.CreateAsync(_connection, options, _chromeProcess, GracefullyCloseChrome).ConfigureAwait(false);
+                var browser = await Browser.CreateAsync(_connection, options.IgnoreHTTPSErrors, options.AppMode, _chromeProcess, GracefullyCloseChrome).ConfigureAwait(false);
```

Comment:
According to pptr SetDefaultViewport is `!appMode` 
https://github.com/kblok/puppeteer/commit/51645932b7dc3c3f3d958deb667c22b3519bcc91#diff-cd756d74c4f724a8d0ebe5f47fff175aR163